tests/e2e-leg-3/labels-annotations/50-assert.yaml

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: vdb-label-ant-cluster1-0
```

Comment:
I thought the `pause`  would prevent the operator to start the pod.

Reply:
The controller for the statefulset isn't paused. So, the pod will still come up. We will never restart vertica in the pod though because the operator is paused.

Reply:
Got it.